orte/tools/orterun/orterun.c

```diff
@@ -2413,8 +2413,8 @@ static void setup_debugger_job(void)
      * to avoid confusing the rest of the system's bookkeeping
      */
     orte_plm_base_create_jobid(debugger);
-    /* set the personality to ORTE */
-    debugger->personality = strdup("orte");
+    /* set the personality to OMPI */
+    debugger->personality = strdup("ompi");
```

Comment:
Because you folks removed the "orte" schizo component in 2.x, so we have to default to the "ompi" one 😄